nginx/templates/default.conf.template

```diff
@@ -0,0 +1,15 @@
+server {
+    listen 80;
+    server_name localhost;
+
+    root /usr/share/nginx/html;
+    index index.html index.htm;
+
+    add_header Content-Security-Policy "script-src 'self' 'unsafe-inline';style-src 'self' 'unsafe-inline';img-src 'self' data:;font-src 'self' https://fonts.gstatic.com;frame-ancestors 'self' ${FRONTEND_URL};block-all-mixed-content";
+
+    add_header Access-Control-Allow-Origin '${FRONTEND_URL}';
```

Comment:
That frond end url can not be fetched dynamically?

Reply:
Not sure, and possibly we may need to add multiple DNS.